packages/syft/src/syft/core/node/common/__init__.py

```diff
@@ -0,0 +1,44 @@
+"""Client AST creation.
+"""
+# stdlib
+from typing import Any
+from typing import Optional
+
+# relative
+from ...node.abstract.node import AbstractNodeClient
+
+
+def create_client_ast(client: Optional[AbstractNodeClient] = None) -> Any:
+    # relative
+    from ....ast import add_classes
+    from ....ast import add_modules
+    from ....ast.globals import Globals
+    from .client import Client
+
+    ast = Globals(client)
+
+    modules = [
+        "syft",
+        "syft.core",
+        "syft.core.node",
+        "syft.core.node.common",
+        "syft.core.node.common.client",
+    ]
+
+    classes = [
+        (
+            "syft.core.node.common.client.Client",
```

Comment:
Do we need an AST entry for Client? Do we need to remotely construct a Client on a remote system from a client itself? Also if this goes into the store, currently while we restrict `.get` we do not restrict arbitrary computation on objects in a store if you know the UID so could a remotely stored client with session token be used by someone to do something bad?